src/lib/Metadata/Reduce.cpp

```diff
@@ -298,6 +299,28 @@ void merge(FriendInfo& I, FriendInfo&& Other)
         I.FriendType = std::move(Other.FriendType);
 }
 
+void merge(AliasInfo& I, AliasInfo&& Other)
+{
+    MRDOCS_ASSERT(canMerge(I, Other));
+    if (I.AliasedSymbol != nullptr)
+        I.AliasedSymbol = std::move(Other.AliasedSymbol);
```

Comment:
This condition needs to be inverted. We only want to overwrite `AliasedSymbol` if it has its default value (`nullptr`).